deployment/hasura/migrations/AerieMerlin/37_constraint_versioning/up.sql

```diff
@@ -0,0 +1,279 @@
+/****** NEW TABLES ******/
+/*-- CONSTRAINT METADATA --*/
+create table constraint_metadata(
+  id integer generated by default as identity,
+
+  name text not null,
+  description text not null default '',
+  public boolean not null default false,
+
+  created_at timestamptz not null default now(),
+  updated_at timestamptz not null default now(),
+
+  owner text,
```

Comment:
Yeah, it will start as the person who created the constraint, but that person can pass their ownership to another user.

The only case the owner would be null is if the user who is the owner gets deleted (the `on delete set null` part of the `constraint_definition_author_exists` fkey).